README.md

```diff
@@ -17,91 +17,168 @@
 </p>
 
 ---
 
-## Getting Started
+# Thor: The VeChainThor Client
 
-VeChainThor is the layer 1 blockchain, highly compatible with EVM<sup>*</sup>, which powers the VeChain ecosystem.
-VeChainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users
-of all sizes and is intended to serve as a foundation for a sustainable and scalable enterprise blockchain ecosystem.
+Thor is the official Golang client for VeChainThor, the public blockchain powering the VeChain ecosystem. VeChainThor is designed for real-world adoption, enabling scalable, low-cost, and sustainable applications.
 
-> VeChainThor is currently up-to-date with the EVM's `shanghai` hard fork,
-> set [evmVersion](https://docs.soliditylang.org/en/latest/using-the-compiler.html#setting-the-evm-version-to-target)
-> to `shanghai` if you are using solidity compiler version `0.8.25` or above.
-___
+> VeChainThor is currently up-to-date with the EVM's `shanghai` hardfork. Set [`evmVersion`](https://docs.soliditylang.org/en/latest/using-the-compiler.html#setting-the-evm-version-to-target) to `shanghai` if you are using Solidity compiler version `0.8.25` or above.
 
-## Documentation
+## Hardware Requirements
 
-- [Build](https://github.com/vechain/thor/blob/master/docs/build.md) - How to build the `thor` binary.
-- [Usage](https://github.com/vechain/thor/blob/master/docs/usage.md) - How to run thor with different configurations.
-- [Hosting a Node](https://github.com/vechain/thor/blob/master/docs/hosting-a-node.md) - Considerations and requirements for hosting a node.
-- [Core Concepts](https://docs.vechain.org/core-concepts) - Core concepts of the VeChainThor blockchain.
-- [API Reference](https://mainnet.vechain.org) - The API reference for the VeChainThor blockchain.
+| Resource  | Validator       | Public Full Node |
+|-----------|-----------------|------------------|
+| CPU       | 4 Core          | 8 Core           |
+| RAM       | 16 GB           | 64 GB            |
+| Bandwidth | 20 Mbit         | 20 Mbit          |
+| Disk      | 500 GB NVMe SSD | 1 TB SSD         |
 
----
+Minimum of 45,000 IOPS required for an approximate 30 hour sync time.
 
-## Community
+## Sync Time
 
-The VeChainThor community can be found on [Discourse](https://vechain.discourse.group/) where you can ask questions,
-voice ideas, and share your projects with other people.
+Sync time from genesis to the latest mainnet block depends on hardware, configuration, and bandwidth.
 
-The VeChain Improvement Proposals (VIPs) repository can be found [here](https://github.com/vechain/VIPs).
+### Validator
 
-To chat with other community members you can join:
+| Build                                         | Sync Time            | AWS SKU                                                      |
+|-----------------------------------------------|----------------------|--------------------------------------------------------------|
```

Comment:
@vanja-vechain this is the setup of minimum validator(Pruner enabled, logDB disabled)?